plugin/evm/ftso_system.go

```diff
@@ -114,28 +114,37 @@ func (f *FTSOSystem) Contracts(hash common.Hash) (FTSOContracts, error) {
 
 	snap := BindEVM(f.blockchain).AtBlock(hash)
 
-	var registryAddress common.Address
-	err := snap.OnContract(f.submitter).Execute(RegistryAddress).Decode(&registryAddress)
-	if err != nil {
-		return FTSOContracts{}, fmt.Errorf("could not get registry address: %w", err)
-	}
-
-	registry := EVMContract{
-		address: registryAddress,
-		abi:     f.abis.Registry,
-	}
-
 	var managerAddress common.Address
-	err = snap.OnContract(f.submitter).Execute(ManagerAddress).Decode(&managerAddress)
+	err := snap.OnContract(f.submitter).Execute(ManagerAddress).Decode(&managerAddress)
 	if err != nil {
 		return FTSOContracts{}, fmt.Errorf("could not get manager address: %w", err)
 	}
 
+	empty := common.Address{}
```

Comment:
```suggestion
        // check if managerAddress is non-empty
	if managerAddress == common.Address{} {
		return FTSOContracts{}, errFTSONotDeployed
	}
```

Reply:
You can't write that, the syntax won't work because of the curly braces.